client/src/components/Form/Elements/parameters.js

```diff
@@ -128,46 +126,6 @@ export default Backbone.View.extend({
         });
     },
 
-    /** Text input field */
-    _fieldText: function (input_def) {
-        // field replaces e.g. a select field
-        const inputClass = input_def.optional && input_def.type === "select" ? Ui.NullableText : Ui.Input;
-        if (
```

Comment:
We should cover for this section in the new component. It preprocesses some of the attributes to funnel a set of special case input definitions into the standardized format expected by the input element.